code/session-1/c++/dapc-f.cpp

```diff
@@ -3,7 +3,7 @@ using namespace std;
 
 signed main() {
     long double x, y;
-    cout << fixed << setprecision(20);
+    cout << setprecision(20);
```

Comment:
Curious, this is a copy of the reference implementation, but good catch.

Reply:
Yes, I got that in my own repository as well. :slightly_smiling_face: And it is as much a "reference implementation" as "just an implementation of one of the jury members who happens to write their template like this by default" :stuck_out_tongue: